### ISSUE TITLE
[BUG] - SCT2 to DXF does nothing

### ISSUE THREAD

**Issue by KSanders7070** [Nikolai558/FE-BUDDY#66]
I select "SCT2 / DXF Conversions"
"SCT2 to DXF"
Source File: ZLC ARTCC.sct2
Output Directory: Desktop
CONVERT

Everything greys out for about 2 seconds and no file is produced on the desktop.

**Comment by Nikolai558** [comment]
I found that in our ZLC Sector file, the problem lies within the [STARS] section. For example, if I removed all but the first star in the section, the dxf file is created like it should. But if you leave all the stars in there, the program "soft crashes" and does not get to the point to write a dxf file. 

If the program is compiled in Visual studio, visual studio either handles this "soft crash" and allows it to continue to write the file. But when the program is compiled with the Powershell script, it just "soft crashes" and does NOT allow the file to be written. 

Further research and help is needed for this issue. I can't comprehend why it is behaving like this for ONLY the "SCT to DXF" functionality.
